sfs/time/drivingfunction.py

```diff
@@ -197,3 +197,65 @@ def apply_delays(signal, delays, fs=None):
     for channel, cdelay in enumerate(delays_samples):
         out[cdelay:cdelay + len(signal), channel] = signal
     return out, offset_samples / fs
+
+
+def wfs_prefilter(dim='2.5D', N=128, fl=50, fu=2000, fs=None, c=None):
+    """Get pre-equalization filter for WFS.
+
+    Rising slope with 3dB/oct ('2.5D') or 6dB/oct ('2D' and '3D').
+    Constant magnitude below fl and above fu.
+    Type 1 linear phase FIR filter of order N.
+    Simple design via "frequency sampling method".
+
+    Parameters
+    ----------
+    N : int, optional
+        Filter order, shall be even. For odd N, N+1 is used.
+    dim : str, optional
+        Dimensionality, must be '2D', '2.5D' or '3D'.
+    fl : int, optional
+        Lower corner frequency in Hertz.
+    fu : int, optional
+        Upper corner frequency in Hertz.
+        (Should be around spatial aliasing limit.)
+    fs : int, optional
+        Sampling frequency in Hertz.
+    c : float, optional
+        Speed of sound.
+
+    Returns
+    -------
+    h : (N+1,) numpy.ndarray
+        Filter taps.
+    delay : float
+        Pre-delay in seconds.
+
+    """
+    N = 2*(int(N + 1)//2)  # for odd N, use N+1 instead
+    if fs is None:
+        fs = defs.fs
+    if c is None:
+        c = defs.c
+
+    numbins = int(N/2 + 1)
```

Comment:
Indeed, `nbins` is as bad as `numbins`. This is of course very much a matter of taste, but I think it's not Pythonic.
Do you know any Python code of the standard library (or in code of any of the Python core contributors) that uses this naming convention?

I understand that `bins` sounds like a list of bins, but it should be immediately clear from its usage that it's not.
I personally wouldn't see this as a problem.

If you want to make it abundantly clear that it is a quantity rather than a list of things, you should call it something like `filtersize`. Or simply `size`, since it's the only "size" thats relevant in this context.
What do you think about this?